.github/workflows/ci.yaml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    env:
+      GOFLAGS: -mod=readonly
+    strategy:
+      max-parallel: 6
+      fail-fast: false
```

Comment:
why fail-fast is false?

Reply:
If a job for one plugin fails, other jobs will immediately fail as well. I thought that this is not the behavior we want. For us, it is better to see all errors in all plugins.

Reply:
got it